test/Microsoft.Windows.CsWin32.Tests/NativeMethods.txt

```diff
@@ -0,0 +1,7 @@
+// Methods (avoid creating more than one per native DLL, as it can cause test churn)
```

Comment:
I'm wondering how with just these few items we end up with many dozens of files, and how adding one more API from the same module is somehow significantly worse than adding an API from another module. 
What is the value of adding these in the first place, and by what rule should we decide on adding more or taking away what we have?